### PR TITLE
Job journal doxygen

### PR DESCRIPTION
**Pull request by gperciva**

### CODE REVIEW

**Comment by gperciva** [comment]
One thing I couldn't figure out in Job:

    signals:
        void loadArchives();

What's this used for?  Is it related to https://github.com/Tarsnap/tarsnap-gui/commit/36e12a7d76b29a6a0902128d923533b441ba906e ?

**Comment by shinnok** [comment]
What you see in 36e12a7 is the old error prone method of loading Job archives from the PersistentStore which resulted in duplicated separate Archive objects and related issues. The new way is to signal using loadArchives() which is connected to TaskManager::loadJobArchives(). If you search for usages of that signal you'll see the various places where it is used.

File: src/persistentmodel/job.h [hunk 1]
```diff
@@ -15,25 +15,39 @@ typedef QSharedPointer<Job> JobPtr;
...
 
 public:
+    //! Constructor.
     explicit Job(QObject *parent = nullptr);
     ~Job();
 
-    QString name() const;
-    void setName(const QString &name);
-
+    //! Returns the job prefix, followed by the archive name.
```
Comment (shinnok) [inline src/persistentmodel/job.h]: Returns JOB_NAME_PREFIX + job name.

File: src/persistentmodel/job.h [hunk 2]
```diff
@@ -69,18 +83,26 @@ class Job : public QObject, public PersistentObject
 
     bool settingHideSymlinks() const;
     void setSettingHideSymlinks(bool settingHideSymlinks);
+    //! @}
 
+    //! Begins performing a backup of this Job.
```
Comment (shinnok) [inline src/persistentmodel/job.h]: This only creates a backupTask() with the properties of this Job, the backup is not initiated yet.

**Comment by gperciva** [comment]
Revised as suggested.

**Pull request merged**